src/realty/domain/realty-address-error-codes.ts

```diff
@@ -0,0 +1,13 @@
+export enum RealtyAddressErrorCodes {
+  STREET_IS_REQUIRED = 'street_is_required',
+  STREET_MUST_BE_LESS_THAN_OR_EQUAL_250 = 'street_must_be_less_than_or_equal_250',
```

Comment:
```suggestion
  STREET_LENGTH_MUST_BE_LESS_THAN_OR_EQUAL_250 = 'street_length_must_be_less_than_or_equal_250',
```

Reply:
na verdade, pro código do erro eu acho que não precisamos especificar o comprimento. Até pq, n é interessante q se o produto resolver mudar o tamanho máximo, a gente tenha que mudar o nome do erro rsrs.
```suggestion
  INVALID_STREET_LENGTH = 'invalid_street_length',
```